torch/testing/_internal/common_methods_invocations.py

```diff
@@ -195,8 +196,8 @@ def __init__(self,
 
     def sample_inputs(self, device, dtype, requires_grad=False):
         low, high = self.domain
-        low = low if low is None else low + self._domain_eps
-        high = high if high is None else high - self._domain_eps
+        low = low if low is None else low
```

Comment:
In a PR that's landing soon this will be changed to no longer use eps but use something like nextafter. Why do you want to change this in this PR?

Reply:
With the change of `floor and ceil`

For functions like `acos` whose domain is `[-1, 1]`,

If we have, `low + self._domain_eps`, with new change, it becomes `0`
and similarly, `high - self._domain_eps` becomes `0`.